umm-spec-lib/test/cmr/umm_spec/test/generate_and_parse.clj

```diff
@@ -105,7 +105,15 @@
       (doseq [target-format tested-collection-formats
               :when (not @failed-atom)
               :let [expected (expected-conversion/convert umm target-format)
+                    expected (update-in-each expected [:Platforms] update-in-each [:Instruments] 
+                               #(assoc % :NumberOfInstruments (let [ct (count (:ComposedOf %))]
+                                                                (when (> ct 0) ct))))
                     actual (xml-round-trip :collection target-format umm)
+                    actual (if (= :iso-smap target-format)
+                             actual
```

Comment:
Talked with Siwei extensively about this. When going from one version to another the :NumberOfInstruments could be generated which doesn't really work within our expected vs actual framework, so this is fine.